generatorv2/lib/utils.rb

```diff
@@ -47,7 +47,7 @@ def additional_json(json)
   def remove_tests(uuid, json)
     json["cases"].each_with_object([]) do |x, acc|
       if x["cases"]
-        acc << { "cases" => remove_tests(uuid, x) }
+        acc << { "cases" => remove_tests(uuid, x), "description" => x["description"] }
       elsif uuid.include?(x["uuid"])
         acc << x
```

Comment:
... with this...

```suggestion
        { "cases" => remove_tests(uuid, x), "description" => x["description"] }
      elsif uuid.include?(x["uuid"])
        x
```

... removes the unconditional work from the conditional branches.

I did not touch the double quotes, though.  Unsure if that is already attended to in another place, if we are doing that.